packages/block-library/src/image/edit.native.js

```diff
@@ -338,7 +338,7 @@ export class ImageEdit extends React.Component {
 						onChange={ this.onSetNewTab }
 					/>
 					{ // eslint-disable-next-line no-undef
-						__DEV__ &&
```

Comment:
So it seems those image added via url don't have an `id` property, maybe it would be clearer to use `id` as a condition instead since it refers to an `id` inside the WordPress Media library,

Reply:
That's the ideal scenario, but we can also get networking errors fetching the image object. Specially those mystery `401` errors that sometimes we get on specific images on self-hosted.

There are some experiments to have (some kind of) authentication on self hosted: https://github.com/wordpress-mobile/WordPressKit-iOS/pull/202 , until then I'd prefer to go with checking if the object is present.